### ISSUE TITLE
Permission denied

### ISSUE THREAD

**Issue by Syko80** [thedatabaseme/awx-ee#14]
Hi @thedatabaseme , I found your website about building AWX EE and then this GitHub.  I think I followed the instructions correctly and when I do my `ansible-builder` command it runs but then I get a 'permission denied' error

Any ideas?

![image](https://user-images.githubusercontent.com/78205923/220416373-72fadcba-98db-40c5-a562-6be0a731c09a.png)

**Comment by thedatabaseme** [comment]
It seems that you user is not allowed to write to /usr/share/ansible/roles. Pleased verify this.

**Comment by Syko80** [comment]
On the Linux machine I am running ansible-builder on or within the container image?  I am just using the base image so would expect everyone or no one to have this issue?

**Comment by thedatabaseme** [comment]
In your linux client were you run the ansible-builder command.

**Comment by Syko80** [comment]
So i think the issue here was that the machine i was using to build the image didnt have ansible installed (??).  I've kicked it off on another server but it seems to be taking a loooong time.  How long did it take for your image to build?

**Comment by thedatabaseme** [comment]
About 12-15 min. But why you don't use the prebuild Images in this repo?

**Comment by Syko80** [comment]
You were right on the permissions stuff and I have used one of your images.  However I wanted to learn how it was all working and your repo has done just that for me.  I am so grateful you shared this stuff!

I needed an EE with azure-cli and terraform so I could do some `shell` commands which are not covered in the azure ansible modules.  I had no idea until I managed to understand your solution and now I've started building my own.  I have even now started looking at the GitHub workflows - it's simple but complex.

Thank you
